plasmapy/formulary/parameters.py

```diff
@@ -612,44 +748,102 @@ def thermal_speed(
+    proportionality determined by the method in which :math:`v_{th}` is
+    calculated and the dimensionality of the system (1D, 2D, 3D).  The
+    :math:`C_o` used for the ``thermal_speed`` calculation is determined from
+    the input arguments ``method`` and ``ndim``, and the values can be seen in
+    the table below:
+
+    .. table:: Values for :math:`C_o`
+       :widths: 2 1 1 1 1
+       :width: 100%
+
+       +--------------+------------+---------------+---------------+---------------+
+       | ↓ **method** | **ndim** → | ``1``         | ``2``         | ``3``         |
+       +--------------+------------+---------------+---------------+---------------+
+       | ``"most_probable"``       | .. math::     | .. math::     | .. math::     |
+       |                           |    0          |    1          |    \sqrt{2}   |
+       +--------------+------------+---------------+---------------+---------------+
+       | ``"rms"``                 | .. math::     | .. math::     | .. math::     |
+       |                           |    1          |    \sqrt{2}   |    \sqrt{3}   |
+       +--------------+------------+---------------+---------------+---------------+
+       | ``"mean_magnitude"``      | .. math::     | .. math::     | .. math::     |
+       |                           |    \sqrt{2/π} |    \sqrt{π/2} |    \sqrt{8/π} |
+       +--------------+------------+---------------+---------------+---------------+
+       | ``"nrl"``                 | .. math::                                     |
+       |                           |    1                                          |
+       +--------------+------------+---------------+---------------+---------------+
+
+    The coefficents can be directly retrieved using
+    `~plasmapy.formulary.parameters.thermal_speed_coefficients`.
+
+        .. rubric:: The Methods
+
+        In the following discussion the Maxwellian distribution
+        :math:`f(\mathbf{v})` is assumed to be 3D, but similar expressions can
+        be given for 1D and 2D.
+
+        - **Most Probable** ``method = "most_probable"``
+
+          This method expresses the thermal speed of the distribution by expressing
+          it as the most probable speed a particle in the distribution may have.
+          To do this we first define another function :math:`g(v)` given by
+
+          .. math::
+             \int_{0}^{\infty} g(v) dv
+                = \int_{-\infty}^{\infty} f(\mathbf{v}) d^3\mathbf{v}
+                \quad \rightarrow \quad
+                g(v) = 4 \pi v^2 f(v)
+
+          then
+
+          .. math::
+             g^{\prime}(v_{th}) = \left.\frac{dg}{dv}\right|_{v_{th}} = 0\\
+             \implies v_{th} = \sqrt{\frac{2 k_B T}{m}}
+
+        - **Root Mean Square** ``method = "rms"``
+
+          This method uses the root mean square to calculate an expression for
+          the thermal speed of the particle distribution, which is given by
+          defines the thermal speed as
+
```

Comment:
```suggestion
          the thermal speed of the particle distribution, which is given by

```
The last clause here appears to be redundant.